earthaccess/auth.py

```diff
@@ -21,6 +24,15 @@
 
 
 logger = logging.getLogger(__name__)
+CLIENT_ID = "ntD0YGC_SM3Bjs-Tnxd7bg"
```

Comment:
Is this the same for all environments (PROD, UAT, and SIT)? If so, then no worries, but if not, then we may need to add a `client_id` attribute to the environment/system instances.

Reply:
I think this is the same for now, though I'm not sure this is ideal for the future. @betolink, can you shed more light on this question?

Reply:
This is a borrowed hard-coded cliend-id from another NSIDC app. I think we should create an earthaccess app in NASA Earthdata. It was only needed because when we interact with the token endpoints or any user endpoint the EDL API requires to know which app is requesting that info (I think this is basically enforcing that any client has to be registered).